### PR TITLE
Fix level requirement for United in Dream

### PR DESCRIPTION
**Pull request by LocalIdentity**

### CODE REVIEW

File: src/Data/Uniques/sword.lua [hunk 1]
```diff
@@ -449,6 +449,7 @@ Variant: Pre 3.0.0
 Variant: Pre 3.16.0
 Variant: Current
 League: Breach
+Requires Level 69
```
Comment (QuickStick123) [inline src/Data/Uniques/sword.lua:452]:
```suggestion
LevelReq: 69
```
The rest of the file seems to be using this format.

**Comment by reruem** [comment]
This is fixed in an upcoming version but not the release version?

**Comment by LocalIdentity** [comment]
This has been fixed for 9 months on the release version

> This is fixed in an upcoming version but not the release version?

image

**Comment by reruem** [comment]
Oh sorry. I loaded this pob [](https://maxroll.gg/poe/build-guides/poison-srs-necromancer/video-guide-pob) and it still said level 38. I didn't realize "All items" for the build are at-the-time copies of the real src items. So I had to delete it from the all items list and re-add it from the src items

**Pull request merged**